troubleshooting_central/backing-up-central-database-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="backing-up-central-database-by-using-the-roxctl-cli"]
+= Backing up Central database by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: backing-up-central-database-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+Central stores information about:
+
+* Activity observed in your clusters.
+
+* Information retrieved from integrated image registries or Scanners.
+
+* {rh-rhacs-first} configuration.
+
+Backing up the Central database is critical to ensure data integrity and system reliability. Regular backups of the database, which contain the necessary configurations, resources, events and certificates, protect against database failures, corruption and accidental data loss.
+
+You can use the `roxctl` CLI to back up and restore the Central database by using the `backup` command. You require an API token or your administrator password to run this command.
```

Comment:
Possible rewrite to avoid "You require":
```suggestion
You can use the `roxctl` CLI to back up and restore the Central database by using the `backup` command. This command requires an API token or your administrator password.
```